.github/workflows/pants.yaml

```diff
@@ -26,14 +26,46 @@ jobs:
         python-version: [3.7]
     steps:
     - uses: actions/checkout@v2
+    - name: Get Pants version
+      id: pants_version
+      run: |
+        # Capture the "pants_version = " line from config.
+        PANTS_VERSION=$(grep -E '^pants_version\s*=' pants.toml)
```

Comment:
Grep is so much easier, lol 😅

Reply:
It's less accurate in that we're pulling out the entire line, and not just the version string. But for invalidation purposes that is fine! As long as no one tries to use that string for other purposes.